integration-tests/bootstrap/bootstrap_test.go

```diff
@@ -0,0 +1,18 @@
+package bootstrap
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Getting Kubernetes version", func() {
+	Context("using kubeconfig", func() {
```

Comment:
Message can be "by discovering kubernetes version"

Please refer `https://github.com/kubernetes-sigs/controller-runtime/blob/v0.1.9/pkg/client/client_test.go`

Reply:
done